### ISSUE TITLE
SNY and YES Network Support

### ISSUE THREAD

**Issue by vendza** [tonywagner/EPlusTV#86]
Please add SNY and Yes Networks to supplement EPlusTV

**Comment by adamef93** [comment]
YES is being combined with MSG+ to create a new service called Gotham Sports. I opened another issue to light the beacon since this will likely break MSG+, so fixing that/adding Gotham Sports would likely include YES

**Comment by m0ngr31** [comment]
I'll wait to add in YES until Gotham lands, but if someone wants to lend me SNY credentials I'll investigate what it'd take

**Comment by m0ngr31** [comment]
YES was added in 4.1.0

**Comment by m0ngr31** [comment]
SNY is included in the MLB.tv integration now
